pkg/metadata/provider_aws.go

```diff
@@ -61,6 +80,19 @@ func (p *ProviderAWS) Extract() ([]byte, error) {
 	return userData, nil
 }
 
+// lookup a value (lookupName) in aws metaservice and store in given fileName
+func awsMetaGet(lookupName string, fileName string, fileMode os.FileMode) {
+	lookupValue, err := awsGet(metaDataURL + lookupName)
+	if err == nil {
+		err = ioutil.WriteFile(path.Join(ConfigPath, fileName), lookupValue, fileMode)
```

Comment:
let's leave it as is for now. I guess it depends on how it will be used and we don't know that yet